scss/abstracts/_variables.scss

```diff
@@ -42,4 +42,17 @@ $gold--90: #472214;
 $gold--70: #E05A00;
 $gold--50: #FF871F;
 $gold--30: #FFA947;
-$gold--10: #FFE7B8;
\ No newline at end of file
+$gold--10: #FFE7B8;
+
+
+$device--sm: 65em;
+// $device--md: 60em;
```

Comment:
Could we uncomment this one?

Reply:
removing for now, i don't actually think we use this size